portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class Comment {
+    public Comment() {
+
+    }
+
+    public Comment(int id, String name, String description, Date postDate) {
```

Comment:
Can make this constructor private so users have to use the builder https://dzone.com/articles/the-builder-pattern-for-class-with-many-constructo